documentation/README_quadruped_master_board.md

```diff
@@ -0,0 +1,96 @@
+# Quadruped master board
```

Comment:
I think this page is outdated. A lot of information was written at the specification phase or are redondant with the already existing github doc. I would remove this one.